src/PSRule.Rules.Azure/rules/Azure.BV.Rule.yaml

```
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+#
+# Validation rules for Azure Backup Vault (BV)
+#
+
+#region Rules
+
+---
+# Synopsis: Ensure immutability is configured to protect backup data.
+apiVersion: github.com/microsoft/PSRule/v1
+kind: Rule
+metadata:
+  name: Azure.BV.Immutable
+  ref: AZR-000398
+  tags:
+    release: GA
+    ruleSet: 2023_09
+    Azure.WAF/pillar: 'Security'
+  labels:
+    Azure.MCSB.v1/control: 'DP-2'
```

This control aligns more closely with _BR-2_.

```suggestion
    Azure.MCSB.v1/control: 'BR-2'
```